components/maskedtextbox/overview.md

```diff
@@ -10,71 +10,104 @@ position: 0
+| `Mask` | `string` | The mask (pattern) that the user has to follow. |
+| `MaskOnFocus` | `bool` | Controls if the mask will show only while the input is focused. When `true` and there is no value, the user will see the [FloatingLabel]({%slug floatinglabel-overview%}) or `Placeholder` (if used) instead of the mask. When there is some value, the mask and value will show. |
+| `Prompt` | `char` <br /> (`_`) | The character that will show in the mask where there is no user input already. |
+| `PromptPlaceholder` | `char?` <br /> (` ` space) | The character that is added to the raw `Value` for places where there is no user input yet. |
 
 ## MaskedTextBox Parameters
 
+See the [MaskedTextBox API Reference](/blazor-ui/api/Telerik.Blazor.Components.TelerikMaskedTextBox) for a full list of parameters, methods and events.
+
 | Parameter | Type and Default value | Description |
-|-----------|------------------------|-------------|
-| `Class`  | `string` | Adds a custom CSS class to the `<div class="k-maskedtextbox">` element. |
-| `Width`  | `string` | Controls the width of the `<input>` element. |
-| `DebounceDelay`  | `int` <br /> (`150`) | Specifies the time in milliseconds between the last typed symbol and the updating of the value. |
-| `Enabled`  | `bool` <br /> (`true`) | Controls if the users can type in the component. |
-| `Id`  | `string` | Renders as the `id` attribute on the `<input />` element, so you can attach a `<label for="">` to the input |
-| `Name`  | `string` | Renders as the `name` attribute of the HTML element. |
-| `Placeholder` | `string` | Renders as the `placeholder` attribute of the HTML element. |
-| `TabIndex` | `int` | Maps to the `tabindex` attribute of the HTML element. You can use it to customize the order in which the inputs in your form focus with the `Tab` key. |
-| `Title` | `string` | Maps to the `title` attribute of the HTML element. You can use it to add a [tooltip]({%slug tooltip-overview%}). |
-| `ValidateOn` | `ValidationEvent` enum <br /> (`Input`) | configures the event that will trigger validation (if validation is enabled). See the [Validation Modes for Simple Inputs]({%slug common-features/input-validation%}#validation-modes-for-simple-inputs) article for more information. |
-| `Value`  | `string` | Get/set the value of the input, can be used for binding. |
+|---|---|---|
+| `AriaDescribedBy` | `string` | The [`aria-describedby` attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby) of the `input`. |
+| `AriaLabel` | `string` | The [`aria-label` attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) of the `input`. |
+| `AriaLabelledBy` | `string` | The [`aria-labelledby` attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) of the `input`. |
+| `Class` | `string` | The custom CSS class of the `<span class="k-maskedtextbox">` element. |
```

Comment:
We may consider general practice for consistent parameters sectioning. For some components we list parameters such as `Class`, `Width`, `Height` in [Styling and Appearance section](https://docs.telerik.com/blazor-ui/components/autocomplete/overview#styling-and-appearance). However, this is not always the case.